doc/reference/reference_lua/config.rst

```diff
@@ -524,6 +563,295 @@ The ``config.storage`` API allows you to interact with a Tarantool-based :ref:`c
 
     Example on GitHub: `tarantool_config_storage <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage>`_
 
+
+..  _config_storage_client_api_reference:
+
+config.storage_client API
+~~~~~~~~~~~~~~~~~~
+
+    The ``config.storage_client`` API allows you to connect to a remote replica set configured as a Tarantool-based :ref:`centralized configuration storage <configuration_etcd>` and interact with it as a key-value storage.
+
+.. module:: config.storage_client
+
+.. _config_storage_client_api_reference_connect:
+
+.. function:: connect(endpoints[, options])
+
+    Connect to a remote config.storage cluster.
+
+    :param table endpoints: table of config.storage instance URIs
+    :param table options: client options to pass. The following options are available:
+
+                           -   ``timeout`` (default: ``nil``, unset) -- connect & call timeout.
+
+    :return:    a <config.storage client> object
+
+    :rtype: table
+
+The example below shows how to connect to a specific config.storage cluster:
+
+..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/config_storage/myapp.lua
+    :language: lua
+    :start-after: config = require
+    :end-at: })
+    :dedent:
+
+The example below shows how to connect to a config storage cluster configured as the :ref:`centralized configuration storage <configuration_etcd>` in the :ref:`config.storage <configuration_reference_config_storage>` option:
+
+..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/config_storage/myapp.lua
+    :language: lua
+    :start-after: function connect_to_configured_storage
+    :end-at: config.storage_client.connect
+    :dedent:
+
+Examples on GitHub: `config_storage <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/centralized_config/instances.enabled/config_storage>`_
+
+.. _config_storage_client_api_reference_put:
+
+.. function:: <config.storage client>:put(path, value)
+
+Put a value by the specified path to remote config.storage.
+
+:param string path: a path to put the value by
+:param string value: a value to put
```

Comment:
Don't know why, but it seems the rendering fails here:

![image](https://github.com/user-attachments/assets/98a6befd-8007-490d-b99d-2061134cd5ba)

Reply:
And in other places too.